package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "vue-typescript-admin-template",
+  "name": "incubator-apisix-dashboard",
   "version": "0.1.0",
   "private": true,
-  "author": "Chong Guo <armourcy@gmail.com>",
+  "author": "apache",
```

Comment:
@moonming  I'm not sure 🤔 who or which community is suitable here.

Reply:
Is this `author` field required?

Reply:
I think we can remove this field

Reply:
ok